NEW_README.md

```diff
@@ -0,0 +1,25 @@
+# Bevy Assets
+
+A repository of cool Bevy projects. If you would like to share what you're working on, submit a PR! Feel free to create new categories where it makes sense.
+
+This repository can be explored in the [Bevy Asset website](https://bevyengine.org/assets/).
+
+## Asset Format
+
+Assets are described in a toml file:
+
+```toml
+name = "Official Bevy Examples"
```

Comment:
This might just be my brain, but I think documenting every field looks better / helps people mentally establish a [documentation] -> [field] pattern.